llvm/test/Instrumentation/MemorySanitizer/PowerPC/kernel-ppc64le.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: opt < %s -S -msan-kernel=1 -passes=msan 2>&1 | FileCheck %s
```

Comment:
Since the directory name contains `PowerPC`, it's unnecessary to repeat `ppc64le`.

You can use `basic-kernel.ll` like SystemZ/ or `kernel.ll`.

Reply:
The other tests in the directory are named this way so I followed the convention.

Reply:
`llvm/test/Instrumentation/MemorySanitizer/PowerPC/vararg-ppc64.ll` does not follow the convention. I think the new test should follow other tests which follow the convention.

Reply:
I see what you mean. Would something like `kernel-64le.ll` work since PowerPC encompasses both 32 and 64 bit and big and little endian?